crypto/merklesignature/merkleSignatureScheme_test.go

```diff
@@ -30,33 +30,25 @@ import (
 	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
-type TestingHashable struct {
-	data []byte
-}
-
-func (s TestingHashable) ToBeHashed() (protocol.HashID, []byte) {
-	return protocol.TestHashable, s.data
-}
-
 func TestSignerCreation(t *testing.T) {
```

Comment:
Most parts of the `merkleSignatureScheme` tests generate falcon keys which utilize the CPU with several go routine.
Do you think it would still help?

Reply:
I ran it locally and it sped up by about 3-4 seconds out of 12.